modules/shortcodes/facebook.php

```diff
@@ -51,7 +55,11 @@
  * @param array $url     Requested URL to be embedded.
  */
 function jetpack_facebook_embed_handler( $matches, $attr, $url ) {
-	if ( false !== strpos( $url, 'video.php' ) || false !== strpos( $url, '/videos/' ) ) {
+	if (
+		false !== strpos( $url, 'video.php' )
+		|| false !== strpos( $url, '/videos/' )
+		|| false !== strpos( $url, '/watch/' )
```

Comment:
Should we be requiring the trailing slash here? 

`https://www.facebook.com/watch?v=2836814009877992` seems to also be valid, but would not match here.

Reply:
Good point. Fixed in f6423c326f81c8e4c558f0d4cb3f5d4c9b0a354d